src/pages/docs/theme/getting-started.mdx

```diff
@@ -21,24 +21,6 @@ export const App = () => (
 **Result:** In this example, the `darkTheme` from Reablocks is applied to your application.
 Simply wrap your component hierarchy with the `ThemeProvider`, and the theme is automatically applied.
 
-### Create a reusable component with a custom theme
-You can also pass theme={} directly to any component, which will override the theme for that component, but not its children.
-This is useful if you want to create a reusable component with a custom theme.
-
-```tsx
-import { tooltipTheme, TooltipTheme, extendComponentTheme } from 'reablocks';
-
-const customTheme = extendComponentTheme<TooltipTheme>(tooltipTheme, {
-  base: 'rounded bg-green-800 text-white font-bold p-3 text-base'
-});
-
-export const CustomTooltip = () => (
-  <Tooltip theme={customTheme} content="Hi there">Hover me</Tooltip>
-);
-```
-
-&nbsp;
-
 ### Extending the Reablocks Theme
```

Comment:
Extending the theme and adding your own tw tokens feel like they should come before overriding specific component themes imo